docs/getting-started/generated-app.md

```diff
@@ -32,9 +32,6 @@ Below are the technologies employed for Node.js and .NET, ensuring efficient, re
 
 - [NestJS](https://nestjs.com/) - A progressive Node.js framework for building efficient, reliable and scalable server-side applications.
 - [Prisma](https://www.prisma.io/) - Next-generation ORM for Node.js and TypeScript.
-- [PostgreSQL](https://www.postgresql.org/) - The world's most advanced open source relational database
```

Comment:
I think we DO need to mention database capabilities here, and add some (all the existing) examples. I don't think we need to remove it at all...